Makefile

```diff
@@ -161,7 +161,7 @@ vet:
 
 .PHONY: generate
 generate: fomantic js css
-	GO111MODULE=on $(GO) generate -mod=vendor $(PACKAGES)
+	GO111MODULE=on $(GO) generate -mod=vendor -tags '$(TAGS)' $(PACKAGES)
```

Comment:
Perhaps it will be faster to have a `$(BINDATA_PACKAGES)` that only contain the list of packages we actually want to generate; otherwise go will check all packages for dependencies and `generate` directives:

(please ignore the warning):

```
$ time go generate ./modules/*
can't load package: package ./modules/indexer: no Go files in /home/gprandi/src/code.gitea.io/gitea/modules/indexer
writing bindata.go
writing bindata.go
writing bindata.go

real    0m20.242s
user    0m14.794s
sys     0m3.443s

$ time go generate ./modules/public/ ./modules/options/ ./modules/templates/
writing bindata.go
writing bindata.go
writing bindata.go

real    0m13.280s
user    0m13.409s
sys     0m2.669s
```

Reply:
That would mean that we can't add generation to other parts of gitea without manually changing the makefile.